src/messages/MMDSOp.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include "msg/Message.h"
+
+class MMDSOp: public Message {
+public:
+ MMDSOp(int t, int version, int compat_version)
+   : Message(t, version, compat_version) {}
```

Comment:
This can be a template so it can take varargs.  See `RefCountedObjectSafe` for an example.